synapse/pipeline.yml

```
@@ -527,3 +527,39 @@ steps:
 #
 #  - label: Trigger webhook
 #    command: "curl -k https://coveralls.io/webhook?repo_token=$COVERALLS_REPO_TOKEN -d \"payload[build_num]=$BUILDKITE_BUILD_NUMBER&payload[status]=done\""
+
+  ################################################################################
+  #
+  # Complement Test Suite
+  #
+  ################################################################################
+
+  - command:
+      # Build a docker image from the checked out Synapse source
+      - "docker build -t matrixdotorg/synapse:latest -f docker/Dockerfile ."
+      # Download Complement's source code
+      - "wget -N https://github.com/matrix-org/complement/archive/master.tar.gz && tar -xzf master.tar.gz"
```

`-N` is redundant (we don't expect to already have a `master.tar.gz`; indeed its presence causes a warning), and since we're already in a list of commands, we may as well use it instead of our own `&&`.

```suggestion
      - "wget https://github.com/matrix-org/complement/archive/master.tar.gz"
      - "tar -xzf master.tar.gz"
```